recipes/cmake/3.x.x/conanfile.py

```diff
@@ -62,7 +62,7 @@ def configure(self):
 
     def requirements(self):
         if self._with_openssl:
-            self.requires("openssl/1.1.1h")
+            self.requires("openssl/1.1.1i")
```

Comment:
Can you also change the recipe to remove
```
self.info.options.with_openssl = self._with_openssl
```
from `package_id` and do
```
self.options.with_openssl = self._with_openssl
```
in `configure`?
Not that it matters much, but it will make the value of `with_openssl` visible to dependencies.

Reply:
I'm not convinced that the use of a property is good here. Why not simply have in `configure(self)`:
```
        if self.options.with_openssl == "auto":
            self.options.with_openssl = (self.settings.os != "Windows")
```
Then we can use `self.options.with_openssl` and remove the `_with_openssl` property

Reply:
The property was needed before you accepted my change, now you can get rid of it

Reply:
I think a property will still be needed when the value of an option depends on the value of a dependency.
I don't know whether `validate` will help here.